docs/docx_templates.md

```diff
@@ -166,7 +166,7 @@ similarly for conditional or repeated columns.
 If you have a few conditional words in a paragraph, you can use `{% if %}`
 in-line in the paragraph. For example:
 
-`{%if user == "self" %}I request{% else %}{{user}} requests{% endif %} the following:`
+`{%if user == "self" %}I request{% else %}{{ user }} requests{% endif %} the following:`
```

Comment:
This wasn't meant to be part of this PR but it's just a small improvement to the other work in this branch.